spt/OrangeBox/modules/vguimatsurfaceDLL.cpp

```diff
@@ -0,0 +1,449 @@
+#include "stdafx.h"
+#include "vguimatsurfaceDLL.hpp"
+#include <SPTLib\memutils.hpp>
+#include <SPTLib\Windows\detoursutils.hpp>
+#include <SPTLib\hooks.hpp>
+#include "..\modules.hpp"
+#include "..\overlay\overlay-renderer.hpp"
+#include "..\cvars.hpp"
+#include "..\..\utils\string_parsing.hpp"
+#include "Color.h"
+#include "const.h"
+#include "..\scripts\srctas_reader.hpp"
+#include "..\..\vgui\vgui_utils.hpp"
+#include "vgui_controls\controls.h"
+#include "..\patterns.hpp"
+#include "..\overlay\portal_camera.hpp"
+#include "..\..\utils\ent_utils.hpp"
+#include "..\..\utils\property_getter.hpp"
+#include "..\module_hooks.hpp"
+
+#ifndef OE
```

Comment:
Either leave this and get rid of the ones in the function bodies or the other way around.